package.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package>
+<package format="2">
```

Comment:
Why is 2?

Reply:
http://wiki.ros.org/catkin/package.xml#Format_2_.28Recommended.29

The most relevant difference to format 1 (the default) is that there is a `<depend>` tag which combines build and run depend.  So this can make the dependency definition much more compact.